benchmarks/conftest.py

```diff
@@ -0,0 +1,71 @@
+from attr import dataclass
+import json
+import pytest
+from typing import List
+
+from ariadne import gql, make_executable_schema, QueryType
+
+
+@dataclass
+class Group:
+    name: str
+    roles: list
+
+
+@dataclass
+class Avatar:
+    size: int
+    url: str
+
+
+@dataclass
+class User:
+    id: int
+    name: str
+    group: Group
+    avatar: List[Avatar]
+
+
+with open("benchmarks/data.json") as f:
+    data = json.load(f)
+
+
+def users_data():
+    for row in data["users"]:
+        yield User(**row)
+
+
+@pytest.fixture
+def raw_data():
+    return data
+
+
+@pytest.fixture
+def raw_data_one_element():
+    return {"users": [data["users"][0]]}
+
+
+@pytest.fixture
+def hydrated_data():
+    return [user for user in users_data()]
+
+
+@pytest.fixture
+def hydrated_data_one_element():
+    for user in users_data():
+        return [user]
+
+
+query = QueryType()
+
+
+@pytest.fixture
+def type_defs():
+    with open("benchmarks/schema.gql", "r") as file:
```

Comment:
Ariadne has build-in function for this: [`load_schema_from_path`](https://ariadnegraphql.org/docs/api-reference#load_schema_from_path)

Reply:
Replaced with load_schema_from_path function